### ISSUE TITLE
Two branches

### ISSUE THREAD

**Issue by cula** [log4js-node/streamroller#30]
Sorry for been away for several months for private affairs.  
I saw in one log4js issue that log4js-node does not upgrade streamroller version because of too many disk writings.
Now there are 2 branches in this project. I want to know, which should I work on?
And looks like the master is deprecated.

**Comment by nomiddlename** [comment]
Yep, there were a few things that we broke in the last few changes to streamroller that I only realised when I tried to use it with log4js - you could only roll files daily (some people roll hourly, or even more frequently), and all the rolling calls were synchronous (which will cause performance problems for people on slow disks, especially if they compress their logs as well). I've been working on fixing those up in the fix-date-rolling branch. I've only got a couple of issues left to fix and then I'll merge it into master.

**Comment by cula** [comment]
> I've only got a couple of issues left to fix 
Could u please open issue tickets about such issues? I'll look for if there is anything I can help.
The only one I know is about birthtimeMs.

**Comment by nomiddlename** [comment]
The things I'm trying to fix are tests that I've broken. I'll create a PR when I'm done and you can review the changes I've made.

**Comment by cula** [comment]
thx
